tensorflow_datasets/video/moving_sequence_test.py

```diff
@@ -0,0 +1,44 @@
+"""Tests for moving_sequence."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from tensorflow_datasets.core import test_utils
+import tensorflow_datasets.video.moving_sequence as ms
+
+
+class MovingSequenceTest(tf.test.TestCase):
+  @test_utils.run_in_graph_and_eager_modes()
+  def test_images_as_moving_sequence(self):
+    h, w = (28, 28)
+    sequence_length = 8
+
+    vh = 1 / (sequence_length)
+    vw = 1 / (2*(sequence_length))
+    image = tf.ones((28, 28, 1), dtype=tf.uint8)
+
+    velocity = tf.constant([vh, vw], dtype=tf.float32)
+    out_size = (h + sequence_length, w + sequence_length)
+    start_position = tf.constant([0, 0], dtype=tf.float32)
+
+    sequence = ms.image_as_moving_sequence(
+        image, start_position=start_position, velocity=velocity,
+        output_size=out_size, sequence_length=sequence_length)
+    sequence = tf.cast(sequence.image_sequence, tf.float32)
+
+    self.assertAllEqual(
```

Comment:
use `self.evaluate` to actually evaluate these Tensors into values in a way that's graph/eager compatible.
Or you can use `as_numpy`